adyenv6core/resources/adyenv6core-items.xml

```diff
@@ -64,6 +64,15 @@
                         </columntype>
                     </persistence>
                 </attribute>
+                <!-- Apple Pay -->
+                <attribute qualifier="adyenApplePayMerchantName" type="java.lang.String">
+                    <description>Apple Pay merchant name</description>
+                    <persistence type="property"/>
+                </attribute>
+                <attribute qualifier="adyenApplePayMerchantIdentifier" type="java.lang.String">
```

Comment:
Lets align this column name to as we are using "id" at other places instead of "identifier". (adyenApplePayMerchantId)